Students/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Newtonsoft.Json;
+using System.Linq;
+
+namespace Students
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var students = JsonParser.readList<Student>("students.json");
```

Comment:
parsing, retrieving answer and printing is happening in a single class/method. Better to separate them